### PR TITLE
1994 ensure routing is working for all profile edit forms

### LINKED ISSUE #1994: Ensure routing is working for all Profile Edit forms
# Description
Ensure each profile form page has routing (can be reached via a url) and that the links on the main Profile page work properly.

Set up pages, with correct routing between them, for:
- Main profile page 
- About Me
- Language Information
- Government Information
- Work Location
- Work Preferences
- Skills and Experience
- Create Experience x 5 and Edit Experience x5 (nearly identical pages, but different urls for each type, and for new vs existing items)

### PR DESCRIPTION
**Pull request by RVany**
Resolve #1994 
# Description
Ensure each profile form page has routing (can be reached via a url) and that the links on the main Profile page work properly.

Set up pages, with correct routing between them, for:
- Main profile page 
- About Me
- Language Information
- Government Information
- Work Location
- Work Preferences
- Skills and Experience
- Create Experience x 5 and Edit Experience x5 (nearly identical pages, but different urls for each type, and for new vs existing items)

### CODE REVIEW

**Comment by RVany** [comment]
ExperienceAndSkill Component added with hard coded Applicant ID.  DiversityEquityInclusionForm is not ready for development yet so Place holder component added.

**Review by esizer — APPROVED** [review]
Awesome! 😎

File: frontend/talentsearch/src/js/components/diversityEquityInclusion/DiversityEquityInclusionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          title: intl.formatMessage({
+            defaultMessage: "Diversity, equity and inclusion",
+            description:
+              "Display Text for Diversity, equity and inclusion Page",
+          }),
+        },
+      ]}
+    >
+      <div data-h2-flex-item="b(1of1)" data-h2-padding="b(top, m)">
+        <p>Comming Soon!</p>
```
Comment (petertgiles) [inline frontend/talentsearch/src/js/components/diversityEquityInclusion/DiversityEquityInclusionForm.tsx:31]: Typo

**Review by petertgiles — CHANGES_REQUESTED** [review]
Nice work!  Just found one small typo.

**Review by petertgiles — APPROVED** [review]

**Comment by RVany** [comment]
> You might want to add a few KB to bundlewatch.

Ok I will do that before merge it.

**Pull request merged**